config/events.rb

```diff
@@ -13,4 +13,10 @@
   # The above will handle an event triggered on the client like `product.new`.
 
   subscribe :send_chat, :to => ChatController, :with_method => :send_chat
+
+  subscribe :player_clicked, :to => LiveController,
+            :with_method => :player_clicked
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.